packages/aragon-cli/package.json

```diff
@@ -71,6 +73,7 @@
     "opn": "^5.3.0",
     "rimraf": "^2.6.2",
     "semver": "^5.4.1",
+    "source-map-support": "^0.5.11",
```

Comment:
I've tried using `@babel/register` instead, but it uses relative paths which my editor cannot open...:
```
 > TypeError: Cannot read property 'links' of undefined
    at console (../../src/commands/ipfs_propagate.js:37:11)
```

Note: `@babel/register` [uses `source-map-support` under the hood](https://github.com/babel/babel/blob/master/packages/babel-register/src/node.js#L16).